### ISSUE TITLE
Add the ability to show  more info for each Questionnaire item

### ISSUE THREAD

**Issue by f-odhiambo** [google/android-fhir#1689]
**Is your feature request related to a problem? Please describe.**
Add the ability to show the info button(this triggers a popup/hint to show more info) in forms

**Describe the solution you'd like**
On click, we should have a pop-up with more information about the field or form being populated 

**Describe alternatives you've considered**
N/A

**Additional context**
1. This should only be text and not image attachments for now. This should be for each line item (question)
2. We could use a field on the questionnaire to be able to store the information required to be displayed 
3. The pop-up could be a modal that enables the user to see more information and once done is cancellable  
4. The view-more text could potentially be stored in the [Questionnaire.item.definition](http://hl7.org/fhir/questionnaire-definitions.html#Questionnaire.item.definition) field

**Would you like to work on the issue?**
Nope.Will request resourcing to work on this 

**Screenshot**
<img width="350" alt="Screenshot 2022-10-25 at 10 00 54" src="https://user-images.githubusercontent.com/4540684/197740237-6cbcc521-5f44-48bc-9b33-9c866da82d71.png">

**Comment by shelaghm** [comment]
@f-odhiambo Thanks for your request. We have a very similar functionality with the help questionnaire item control https://github.com/google/android-fhir/issues/1528 do you think this could work for your use case?

**Comment by HenryRae** [comment]
@shelaghm possible to have the help button next to the "question" rather than 'instructions' text field in #1528. For cases where the question doesn't have instructions?

**Comment by shelaghm** [comment]
@HenryRae Yes, that's an option. We're hoping to have one location that can stay consistent across so it is easier to implement. See screenshot below with options. Option 1 could still work if there's no instructions as it will just display the help button below the question. 
Do you have a perspective on which option you'd prefer between 1 and 2 in screenshot below? 
I find option 3 (fixed location) to make the icon look a bit disconnected from the content itself. 
<img width="1028" alt="Screen Shot 2022-11-02 at 11 07 07" src="https://user-images.githubusercontent.com/86378921/199567946-f692068c-b6d4-45c1-9e30-b2f4c4b49fa6.png">

**Comment by HenryRae** [comment]
@shelaghm I think a user can easily relate the info action to a question with option 2.

**Comment by shelaghm** [comment]
@HenryRae Thanks for your feedback.

**Comment by HenryRae** [comment]
maybe another thing to consider is if we can include beyond text in them i.e maybe show images in them as well. what are your thoughts on that @shelaghm

**Comment by shelaghm** [comment]
@HenryRae Oh, interesting idea. What's an example of when you'd like to include an image? Do you include images in your apps today?

I imagine it could be really useful for showing visual instructions, especially if it isn't something they do very often. Hearing your specific examples will be really helpful.

**Comment by shelaghm** [comment]
@HenryRae that is clear. Yes, I think for some CHWs if they are doing the same task everyday it is likely that they don't need to always see the visual instructions, and hiding it inside the `help` button can make it available to reference when needed without cluttering the page for all users. 

@jingtang10 are there any engineering constraints for embedding images inside the `help` component?
